torch/_subclasses/fake_tensor.py

```diff
@@ -582,7 +582,7 @@ def nonzero(fake_mode, func, arg):
             # have an empty range which makes things go explodey.  We also
             # don't allow for 2 because that would specialize the unbacked
```

Comment:
is this comment stale then @ezyang

Reply:
No, the fix here brings the code in line with the comment.